index.d.ts

```diff
@@ -217,7 +217,7 @@ interface LoginOptions {
  */
 interface LogoutOptions {
   /**
-   *  URL to returnTo after logout, overrides the Default in {@link ConfigParams.routes.postLogoutRedirect routes.postLogoutRedirect}
+   *  URL to returnTo after logout, overrides the Default in {@link ConfigParams.routes routes.postLogoutRedirect}
```

Comment:
Parameters in a nested object don't get id's so it's not possible to link to them, so instead changes this to link to `Config.routes` so that it does resolve